docs/architecture-decisions/2023-06-06_frontend-templates.md

```diff
@@ -0,0 +1,133 @@
+
+> This approach can be explored and maintained without overlapping with main proposal on frontend templates.
+
+### Addendum 2. Website for choosing preferred frontend and backend repositories.
+
+> TLDR: The approach proposes a website to enhance the discoverability of official and community-based algokit templates, thereby incentivizing template builders to create and maintain their own templates.
+
+This orthogonal approach proposes to improve discoverability of official and community based algokit templates by providing a simple static website. The website can consist of minimalistic UI components for picking preffered backend, frontend and then a `Generate` button that will output copy-pasteable algokit CLI commands to spin up a project with the selected templates.
+
+As a specific example, the website can be hosted on [AwesomeAlgo](https://awesomealgo.com) website, thus ensuring that this is an open-source community maintaned entrypoint for discovering and using algokit templates. Removing the need and maintenance overhead on our teams to maintain it as official resource.
+
+Lastly, community template builders will get a platform to increase discoverability of their templates and further incentivise them to build and maintain them. While developers using the templates can support creators of templates by donating to their projects (a simple tipping mechanism for Algo and ASAs can be embedded into the website) or by contributing to the templates themselves.
+
+> This approach can be explored and maintained without overlapping with main proposal on frontend templates.
+
+---
+
+## Final decision
+
+To be decided
+
+> The final decision is being finalized but the team is currently leaning in favor towards the proposal outlined in the Architecture Decision Record. Hence, this revision excluded the details of other options considered and adds more decision specific details to the approach with having separate self-sufficient frontend templates.
+
+## Open questions
+
+- Python typed clients are not going to be too relevant for scenarios where they need to be integrated with frontends given that in majority of cases a developer would prefer a TS typed client. Hence, how do we ensure that user gets a clear information and indication on when to use Python typed clients vs TS typed clients?
+- - Would we want to introduce a notion of non smart contract based backend templates that can be used to plug in the python typed clients and spin up servers that can be used to build APIs that interact with the smart contracts?
+
+## Next steps
+
+After the final decision is made, the action items necessary to implement the described proposal can be outlined as follows:
+
+1. **Design and Development of Independent Frontend Templates**: This involves selecting appropriate technology stacks and building out the templates. Each of these templates should be able to work with the generated typed client from the backend.
```

Comment:
I think we are only doing 1 rather than multiple, obviously community contributions can create separate ones though.

Reply:
Tweaked to explicitly state that its a single repo